src/content.tsx

```diff
@@ -7,9 +7,17 @@ import { storage } from 'webextension-polyfill'
 import { Tako, TakoProvider } from './components/Tako'
 import { TokenPrompt } from './components/TokenPrompt'
 import { useStore } from './store'
-import { waitForElement } from './waitForElement'
+import { onElementRemoval, waitForElement } from './waitForElement'
 
 const start = async () => {
+  if (!isRepoRoot() || !isRepoTree()) {
+    return
+  }
```

Comment:
What about being even more defensive?

```suggestion
  if (!isRepoRoot() || !isRepoTree() || !document.querySelector('.tako')) {
    return
  }
```

Reply:
Ah, great idea!